Support more i18n keys in messages_ru.properties for login theme

petrov9 opened the pull request:
Current messages_ru.properties file is a full copy of base/messages_en.properties file but with translated values

Closes #40108

petrov9 committed:
Support more i18n keys in messages_ru.properties for login theme

Current messages_ru.properties file is a full copy of base/messages_en.properties file but with translated values

Closes #40108

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented:
> > How to "Find a reviewer that is a native speaker in that language"
> 
> Usually people who submit a translation (you) are native speakers. People than ask a friend or co-worker to review the pull request. You can also reach out to the keycloak-user mailing list or the Keycloak slack channel to ask.
> 
> We don't have a set of language maintainer for Russian yet - see [#9270 (comment)](https://github.com/keycloak/keycloak/discussions/9270#discussioncomment-13197107)

Got it, thanks

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -3,20 +3,27 @@ doRegister=Регистрация
+doTryAnotherWay=Попробуйте другой способ
+doConfirmDelete=Подтвердите удаление
+errorDeletingAccount=Произошла ошибка при удалении аккаунта
+deletingAccountForbidden=У вас недостаточно прав для удаления собственной учетной записи, обратитесь к администратору.
```
нужно ли упоминание "собственной" ?

petrov9 replied:
Согласен, убрал

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -3,20 +3,27 @@ doRegister=Регистрация
 kerberosNotConfigured=Kerberos не сконфигурирован
 kerberosNotConfiguredTitle=Kerberos не сконфигурирован
-bypassKerberosDetail=Либо вы не вошли в систему с помощью Kerberos, либо ваш браузер не настроен для входа в систему Kerberos. Пожалуйста, нажмите кнопку ''Продолжить'' для входа в с помощью других средств
+bypassKerberosDetail=Либо вы не вошли в систему с помощью Kerberos, либо ваш браузер не настроен для входа в систему Kerberos. Пожалуйста, нажмите кнопку продолжить для входа с помощью других средств
```
Можно выделить слово "продолжить" или с заглавной буквы или оставить кавычки или как-то ещё.

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -25,31 +32,57 @@ realmChoice=Realm
 termsTitle=Условия и положения
-termsTitleHtml=Условия и положения
 termsText=<p>Условия и положения должны быть определены</p>
+termsPlainText=Условия и положения подлежат определению.
```
можно для однообразия как в 71-й - "Условия и положения должны быть определены"

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -72,46 +107,92 @@ region=Регион
 postal_code=Почтовый индекс
 country=Страна
 emailVerified=E-mail подтвержден
+website=Веб-страница
```
Можно крупнее - "Веб-сайт"

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -131,49 +212,96 @@ role_manage-clients=Управление клиентами
 accountTemporarilyDisabledMessageTotp=Неверный код аутентификатора.
 accountPermanentlyDisabledMessageTotp=Неверный код аутентификатора.
 expiredCodeMessage=Вход просрочен по таймауту. Пожалуйста, войдите снова.
+expiredActionMessage=Действие истекло. Пожалуйста, продолжите вход сейчас.
```
можно "Сеанс завершён" или "Сессия завершена"

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -131,49 +212,96 @@ role_manage-clients=Управление клиентами
 expiredCodeMessage=Вход просрочен по таймауту. Пожалуйста, войдите снова.
+expiredActionMessage=Действие истекло. Пожалуйста, продолжите вход сейчас.
+expiredActionTokenNoSessionMessage=Действие истекло.
+expiredActionTokenSessionExistsMessage=Действие истекло. Пожалуйста, начните снова.
```
можно "Сеанс завершён" или "Сессия завершена"

pasternake commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -131,49 +212,96 @@ role_manage-clients=Управление клиентами
 notMatchPasswordMessage=Пароли не совпадают.
 
+error-invalid-value=Недопустимое значение.
+error-invalid-blank=Укажите стоимость.
```
"значение" ?

petrov9 replied:
готово

petrov9 committed:
Support more i18n keys in messages_ru.properties for login theme. Values clarification

Current messages_ru.properties file is a full copy of base/messages_en.properties file but with translated values

Closes #40108

Signed-off-by: Anton Petrov <petrov9810@gmail.com>

petrov9 commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -25,31 +32,57 @@ realmChoice=Realm
 unknownUser=Неизвестный пользователь
 loginTotpTitle=Настройка мобильного аутентификатора
 loginProfileTitle=Обновление информации учетной записи
+loginIdpReviewProfileTitle=Обновить информацию об учетной записи
```
Сделал

petrov9 commented on themes/src/main/resources-community/theme/base/login/messages/messages_ru.properties:
```diff
@@ -131,49 +212,96 @@ role_manage-clients=Управление клиентами
 
+error-invalid-value=Недопустимое значение.
+error-invalid-blank=Укажите стоимость.
+error-empty=Укажите стоимость.
```
Спасибо, поправил

petrov9 commented:
@ahus1 could you please take a look, I got approval from a native speaker

ahus1 commented:
@pasternake  / @petrov9 - thank you for your collaboration on the translation. Let me know if you want to collaborate permanently on the Russian translation, and if I should set up Weblate for the two of you or other two users.

pasternake commented:
> @pasternake / @petrov9 - thank you for your collaboration on the translation. Let me know if you want to collaborate permanently on the Russian translation, and if I should set up Weblate for the two of you or other two users.

I'm ready for collaboration.

petrov9 commented:
@ahus1 We are interested in it, and discussed it with @pasternake but neither of us understands what exactly we should do as translate collaborators using Weblate?
